app/views/account/users/show.html.slim

```diff
@@ -0,0 +1,29 @@
+.container
+  h1.title
+    | User info:
+  .show-box
+    .user-avatar-show
+      - if @user.avatar?
+        = image_tag(@user.avatar.url, alt: 'Image', class: 'user-avatar show-user')
+      - else
+        = image_tag('default_a.png', alt: 'Image', class: 'user-avatar show-user')
```

Comment:
met this second time, makes sense to move that to the helper(https://github.com/IhorMoskvyn/Bookshelf/pull/7/files#r658843236)

Reply:
done